spec/services/edge_cache/bust_spec.rb

```diff
@@ -4,6 +4,10 @@
   let(:user) { create(:user) }
   let(:path) { "/#{user.username}" }
 
+  it "defines TIMEFRAMES" do
+    expect(described_class.const_defined?(:TIMEFRAMES)).to be true
+  end
```

Comment:
I would use `Timecop` to test that by moving forward or backwards in time those timeframes adjust to the current time inside the Timecop block, just to be on the safe side (I think testing one of them is enough)

Reply:
Good idea! I gave it a try, let me know what you think @rhymes. I understand your point, but I'm less sure that I implemented the way you described.